provider/cloudflare/cloudflare.go

```diff
@@ -634,12 +682,16 @@ func (p *CloudFlareProvider) listDNSRecordsWithAutoPagination(ctx context.Contex
 	return records, nil
 }
 
+func getCustomHostnameIndex(ch cloudflare.CustomHostname) CustomHostnameIndex {
```

Comment:
NewCustomHostnameIndex()

Reply:
updated

Reply:
sry should be lowercase aka package private

Reply:
updated